packages/wonder-blocks-birthday-picker/src/components/__tests__/birthday-picker.test.tsx

```diff
@@ -63,8 +63,7 @@ describe("BirthdayPicker", () => {
 
         it("renders with a valid default value", async () => {
             // Arrange
-            const date = moment(today);
-            const defaultValue = date.format("YYYY-MM-DD");
+            const defaultValue = Temporal.Now.plainDateISO().toString();
```

Comment:
These test failures are interesting...they seem to be off by a month? I saw your comment that Temporal has 1-based indexing for months, but these are referenced by date string and text content. Any ideas?

Reply:
I'm actually really thrown off by this, cuz all these tests pass locally :-( I'm gonna have to do some digging

Reply:
Finally got the test failures fixed!